apps/chat/src/components/Chat/Publish/PublicationHandler/PublicationHandlerFooter.tsx

```diff
@@ -0,0 +1,448 @@
+import { useAppDispatch, useAppSelector } from '@/src/store/hooks';
+import { ModelsSelectors } from '@/src/store/models/models.selectors';
+import { EDITED_FOLDER_NAME_KEY } from '@/src/store/publication/publication.types';
+import {
+  ConversationsSelectors,
+  PromptsSelectors,
+  PublicationSelectors,
+} from '@/src/store/selectors';
+
+import {
+  MAX_PUBLICATION_AUTHOR_LENGTH,
+  NA_VERSION,
+} from '@/src/constants/publication';
+
+import { IconButton } from '@/src/components/Common/IconButton';
+import { Tooltip } from '@/src/components/Common/Tooltip';
+
+import { FeatureType } from '@epam/ai-dial-shared';
+import uniq from 'lodash-es/uniq';
+
+const allEditedFoldersAreValid = (obj: unknown) => {
+  for (const key in obj as Record<string, string>) {
+    const value = (obj as Record<string, string>)[key];
+
+    if (typeof value === 'object' && value !== null) {
+      if (
+        EDITED_FOLDER_NAME_KEY in value &&
+        (!value[EDITED_FOLDER_NAME_KEY] ||
+          prepareEntityName(value[EDITED_FOLDER_NAME_KEY] as string).trim() ===
+            '')
+      ) {
+        return false;
+      }
+
+      if (!allEditedFoldersAreValid(value)) {
+        return false;
+      }
+    }
+  }
+
+  return true;
+};
+interface Props {
+  publication: Publication;
+  publicationAuthor: string;
+  onUpdateRequest: () => void;
+}
+
+export const PublicationHandlerFooter = ({
+  publication,
+  publicationAuthor,
+  onUpdateRequest,
+}: Props) => {
+  const { t } = useTranslation(Translation.Chat);
+
+  const files = useAppSelector(FilesSelectors.selectFiles);
+  const prompts = useAppSelector(PromptsSelectors.selectPrompts);
+  const conversations = useAppSelector(
+    ConversationsSelectors.selectConversations,
+  );
+  const applications = useAppSelector(
+    ModelsSelectors.selectPublishRequestModels,
+  );
+  const resourcesToReview = useAppSelector((state) =>
+    PublicationSelectors.selectResourcesToReviewByPublicationUrl(
+      state,
+      publication.url,
+    ),
+  );
+  const isEditMode = useAppSelector(PublicationSelectors.selectIsEditMode);
+  const entitiesEditState = useAppSelector(
+    PublicationSelectors.selectEntitiesEditState,
+  );
+  const foldersEditState = useAppSelector(
+    PublicationSelectors.selectFoldersEditState,
+  );
+  const displayAuthorEditState = useAppSelector(
+    PublicationSelectors.selectDisplayAuthorEditState,
+  );
+  const itemsToPublish = useAppSelector(
+    PublicationSelectors.selectSelectedItemsToPublish,
+  );
+
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    // reset edit mode state when publication changes
+    dispatch(PublicationActions.setIsEditMode(false));
+  }, [dispatch, publication.url]);
+
+  useEffect(() => {
+    dispatch(
+      PublicationActions.setEditModeState({
+        editState: getDefaultAllEditEntities(publication.resources),
+        rules: publication.rules,
+        displayAuthor: publication.displayAuthor ?? publicationAuthor,
```

Comment:
```suggestion
        displayAuthor: publication.displayAuthor,
```

TODO was removed from PublicationHandler
lets keep it synced